.eslintrc.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 				selector: 'CallExpression[callee.name="deprecated"] Property[key.name="version"][value.value=/' + majorMinorRegExp + '/]',
 				message: 'Deprecated functions must be removed before releasing this version.',
 			},
+			{
+				selector: 'JSXIdentifier[name=/^(path|svg)$/]',
```

Comment:
Should we also add `<g>` and `<polygon>` that we already have componentized as well?

Reply:
Yes, sure. 

> The eslint rule is too aggressive at the moment, it also catches props named "path"

As Riad mentioned, this rule needs to be updated anyways.